internal/datagen/generator.go

```diff
@@ -1,9 +1,9 @@
 package datagen
 
 import (
+	"crypto/rand"
```

Comment:
It's much slower and we don't need crypto-level randomness. Better use `math/rand/v2`.

Reply:
In `math/rand/v2` there is an analog using the ChaCha8-based cryptographically strong random number generator, so you need to create a generator variable and run Read from it. Is this suitable for us? Everywhere recommended using exactly "crypto/rand.Read" instead of the deprecated `math/rand.Read`.

Reply:
We can't answer this till it's tested in some way, what's the speed of old rand/new rand/crypto rand? Create a benchmark, try it. This thing just needs to push a lot of gobbledygook into buffer and do it fast.

Reply:
```
goos: linux
goarch: amd64
pkg: github.com/nspcc-dev/xk6-neofs/internal/datagen
cpu: AMD Ryzen 7 PRO 4750U with Radeon Graphics
BenchmarkMathRandRead-16          602894              1997 ns/op
BenchmarkMathRandV2Read-16       1664827               734.3 ns/op
BenchmarkCryptoRandRead-16        371931              3280 ns/op
```
`math/rand/v2` wins in terms of speed, I will use it.

Reply:
In `math/rand/v2` func `ChaCha8.Read` added in Go 1.23...

Reply:
OK, add a note to #94 (maybe some patch as well).